Description: Fix INET_DIST_INTERFACE option
Author: Ashish SHUKLA <ashish.is@lostca.se>
Origin: upstream
Bug: https://github.com/processone/ejabberd/pull/1869
Applied-Upstream: 17.07+1
Reviewed-by: Mathias Ertl <mati@jabber.at>
Last-Update: 2017-08-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff --git a/ejabberdctl.template b/ejabberdctl.template
index aa2353cc83..efda14ffb5 100755
--- a/ejabberdctl.template
+++ b/ejabberdctl.template
@@ -69,7 +69,7 @@ fi
 if [ "$INET_DIST_INTERFACE" != "" ] ; then
     INET_DIST_INTERFACE2=$("$ERL" -noshell -eval 'case inet:parse_address("'$INET_DIST_INTERFACE'") of {ok,IP} -> io:format("~p",[IP]); _ -> ok end.' -s erlang halt)
     if [ "$INET_DIST_INTERFACE2" != "" ] ; then
-        ERLANG_OPTS="$ERLANG_OPTS -kernel inet_dist_use_interface \"$INET_DIST_INTERFACE2\""
+        ERLANG_OPTS="$ERLANG_OPTS -kernel inet_dist_use_interface $INET_DIST_INTERFACE2"
     fi
 fi
 ERL_LIBS={{libdir}}
